kong/cmd/config.lua

```diff
@@ -97,6 +97,16 @@ local function execute(args)
 
       log("import successful")
 
+      -- send anonymous report if reporting is not disabled
+      if conf.anonymous_reports then
+        local kong_reports = require "kong.reports"
+        kong_reports.configure_ping(conf)
+        kong_reports.toggle(true)
+
+        local report = { _format_version = err_or_ver }
+        kong_reports.send("dbless-import", report)
```

Comment:
This mode is not db-less, since it imports the declarative config file in the DB itself; this signal should be something like `db-import`